ecosystem/README_zh-CN.md

```diff
@@ -244,3 +244,54 @@ LlamaIndex 是一个用于构建上下文增强型 LLM 应用程序的框架。
 它选择 ollama 作为 LLM 推理引擎。你可以在[入门教程（本地模型）](<(https://docs.llamaindex.ai/en/stable/getting_started/starter_example_local/)>)中找到示例。
 
 因此，如果能够按照 [ollama 章节](#ollama)使用 ollama 部署浦语模型，你就可以顺利地将浦语模型集成到 LlamaIndex 中。
+
+### [LazyLLM](https://github.com/LazyAGI/LazyLLM)
+
+LazyLLM 是一个的低代码构建多 Agent 大模型应用的开发工具，相比于 LangChain 和 LLamaIndex，其具有极高的灵活性和易用性。
+
+当您依次通过 `pip3 install lazyllm` 和 `lazyllm install standard` 安装了 LazyLLM 之后, 您可以使用如下代码以极低的成本，基于 InternLM 搭建 chatbots，无论推理还是微调，您都无需考虑对话模型的特殊 token（如`<|im_start|>system`和`<|im_end|>`等 ）。不用担心没有权重文件，只要您能联网，下面的代码将会自动帮您下载权重文件并部署服务，您只需尽情享受 LazyLLM 给您带来的便利。
+
+```python
+from lazyllm import TrainableModule, WebModule
+m = TrainableModule('internlm2_5-7b-chat')
+# will launch a chatbot server
+WebModule(m).start().wait()
+```
+
+如果您需要进一步微调模型，可以参考如下代码。当 `TrainableModule` 的 `trainset` (数据集需下载到本地，例如：[
+alpaca_gpt4_zh](https://huggingface.co/datasets/llamafactory/alpaca_gpt4_zh))被设置之后，在调用 `WebModule` 的 `update` 函数时，会自动微调 `TrainableModule`，然后对 `TrainableModule` 和 `WebModule` 分别进行部署。
+
+```python
+from lazyllm import TrainableModule, WebModule
+m = TrainableModule('internlm2-chat-7b').trainset('/patt/to/your_data.json').mode('finetune')
```

Comment:
Using llamafactory backend to do internlm2.5 sft. 

error is
![LesUdwiEHe](https://github.com/user-attachments/assets/281afb3f-acf6-4362-9ecd-ee3dbe98f9db)

It seems llamafactory do not have a good support of internlm2.5

Reply:
Indeed, the version of LLaMA-Factory used in the current release of lazyllm does not support this 2.5 version of the model. But the latest LLaMA-Factory supported.